roles/openshift_examples/examples-sync.sh

```diff
@@ -5,7 +5,7 @@
 #
 # This script should be run from openshift-ansible/roles/openshift_examples
 
-XPAAS_VERSION=ose-v1.4.7
+XPAAS_VERSION=ose-v1.4.8
```

Comment:
Unfortunately, our first tag didn't hold, so this is actually __ose-v1.4.8-1__ (notice the __-1__)